### PR TITLE
Week13-WeatherApp

### PR DESCRIPTION
**Pull request by AravindSirivelli**
Node JS Weather App

1. Weather Data of a City     
2. Weather Data of Multiple Cities 
3. Paginated response for Weather Data of Multiple Cities
4. Detailed Forecast for the next X days

### CODE REVIEW

File: Week-13/WeatherApp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+# Dependency directories
+node_modules/
+
+# Optional npm cache directory
+.npm
+
+# dotenv environment variable files
+.env
```
Comment (vatsalyaagarwal) [inline Week-13/WeatherApp/.gitignore:9]: create a sample env file and commit

File: Week-13/WeatherApp/utils/response.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const GenerateSuccessRespone = (body) => {
+    return {
+        "code": 200,
+        "message": "SUCCESS",
+        "data": body
+    }
+}
+const GenerateErrorRespone = (body) => {
+    return {
+        "code": 500,
```
Comment (vatsalyaagarwal) [inline Week-13/WeatherApp/utils/response.js:10]: use the error code from the weather api

**Review by vatsalyaagarwal — APPROVED** [review]

File: Week-13/WeatherApp/.gitignore (hunk 1 above)
Comment (AravindSirivelli) [reply Week-13/WeatherApp/.gitignore:9]: Added ENV Sample and committed

File: Week-13/WeatherApp/utils/response.js (hunk 2 above)
Comment (AravindSirivelli) [reply Week-13/WeatherApp/utils/response.js:10]: The "GenerateErrorRespone" Method returns the JSON output when an Internal Error Occurs within the API (catch). Modified GenerateResponse method to return the Success/Error Code sent by Weather API. Pushed the changes

**Pull request merged**